llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -5368,7 +5369,7 @@ BoUpSLP::findReusedOrderedScalars(const BoUpSLP::TreeEntry &TE,
   // patterns.
   SmallVector<Value *> GatheredScalars(TE.Scalars.begin(), TE.Scalars.end());
   Type *ScalarTy = GatheredScalars.front()->getType();
-  int NumScalars = GatheredScalars.size();
+  uint64_t NumScalars = GatheredScalars.size();
```

Comment:
supernit: size_t?

Reply:
You could probably knock this out with a regex, `s/uint64_t Num(.*)\.size();/size_t Num\1.size();/`